RecoCTPPS/PixelLocal/interface/RPixDetClusterizer.h

```diff
@@ -0,0 +1,122 @@
+/**********************************************************************
+ *
+ * Author: F.Ferro - INFN Genova
+ * September 2016
+ *
+ **********************************************************************/
+#ifndef RecoCTPPS_PixelCluster_DET_CLUSTERIZER_H
+#define RecoCTPPS_PixelCluster_DET_CLUSTERIZER_H
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+
+#include "DataFormats/CTPPSDigi/interface/CTPPSPixelDigi.h"
+#include "DataFormats/CTPPSDigi/interface/CTPPSPixelDigiCollection.h"
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelCluster.h"
+
+#include "CondFormats/CTPPSReadoutObjects/interface/CTPPSPixelGainCalibrations.h"
+#include "RecoCTPPS/PixelLocal/interface/CTPPSPixelGainCalibrationDBService.h"
+
+#include "CondFormats/CTPPSReadoutObjects/interface/CTPPSPixelAnalysisMask.h"
+
+#include <vector>
+#include <set>
+
+class RPixCalibDigi : public CTPPSPixelDigi {
+
+public:
+
+RPixCalibDigi(unsigned char row, unsigned char col, unsigned short adc, unsigned short ele) : CTPPSPixelDigi(row,col,adc){
+    electrons_ = ele;
+  }
+
+RPixCalibDigi() : CTPPSPixelDigi(){}
```

Comment:
the default constructor does not initialize the ```electrons_``` data member; it is OK if the code never uses default-constructed RPixCalibDigi.
Is it the case?
If not, the initialization should be made.